invenio_app_ils/locations/schemas/locations/location-v1.0.0.json

```diff
@@ -34,6 +34,110 @@
     "notes": {
       "type": "string",
       "title": "Extra notes"
+    },
+    "opening_weekdays": {
```

Comment:
In the end, the last discussion we had, was it to move this to another record in the end or not?
The drawback is that if we change the name of the location, or anything else, the code will run... Even if it should not change any loan right?

Reply:
Yes, the code will run anytime a change is detected. It may be possible to detect what change has occurred using the signal, but personally I don't think it is worth it.